spec/features/categories/user_can_delete_category_from_index_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe "when user visits categories index page" do
+  scenario "they can click on delete and category is deleted" do
+
+    category1 = Category.create(title: "Technology")
```

Comment:
I realized today that I need to do some refactoring in my tests for this exact reason. I am calling to much data where I dont need it.